coldfront/core/group/tasks.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import Group
+from coldfront.core.user.models import User
+from coldfront.core.project.models import (
+    Project,
+    ProjectUser,
+    ProjectUserRoleChoice,
+    ProjectUserStatusChoice,
+)
+
+
+def grant_usersupport_global_project_manager() -> None:
+    group_name = "RIS-UserSupport"
+    group = Group.objects.filter(name=group_name).first()
+
+    if not group:
+        return
+
+    all_projects = Project.objects.all()
+    group_users = User.objects.filter(groups=group)
+    role_choice = ProjectUserRoleChoice.objects.filter(name="Manager").first()
```

Comment:
@gunnarwustl I would retrieve the role_choice and status choice before all_projects and groups_users.

Reply:
Is there a reason you would opt to do this beyond simple stylistic choice?